packages/datadog-plugin-http/test/server.spec.js

```diff
@@ -193,6 +193,34 @@ describe('Plugin', () => {
       })
     })
 
+    describe('with a `server` configuration', () => {
+      beforeEach(() => {
+        return agent.load('http', { client: false, server: {} })
+          .then(() => {
+            http = require('http')
+          })
+      })
+
+      beforeEach(done => {
+        const server = new http.Server(listener)
+        appListener = server
+          .listen(port, 'localhost', () => done())
+      })
+
+      // see https://github.com/DataDog/dd-trace-js/issues/2453
+      it('should not have disabled tracing', (done) => {
+        const spy = sinon.spy(() => {})
+        agent.use(spy)
```

Comment:
This returns a promise that is resolved when its callback has been called, so adding an assertion in `use()` would remove the need for a timer.